### PR TITLE
Change record media-body table layout on mobile

### PR DESCRIPTION
**Pull request by maccabeelevine**
On reasonably common phones (testing this on an iPhone 14 in BrowserStack), the use of a Bootstrap media component for the core record data means that the actual table only has ~2/3 of the screen, and its headings take up almost another third, so the actual core data values have just over 1/3 of the screen width.  This means wrapping after just a few words, sometimes even one word, especially for long / scientific / academic terms.

Screenshot 2024-04-29 123812

This PR effectively creates a single-column table at small screen widths, so the table values have full use of use of the media-body width.

Screenshot 2024-04-29 123902

### CODE REVIEW

File: themes/bootstrap3/less/components/record.less [hunk 1]
```diff
@@ -44,6 +44,24 @@
     margin-top: 1rem;
   }
 }
+
+@media (max-width: 500px) {
```
Comment (maccabeelevine) [inline themes/bootstrap3/less/components/record.less:48]: Arguably these styles could be part of the previous media query, but I don't think the layout change is needed in the 500-800px range.  The actual 500px number is just a suggestion.

**Comment by maccabeelevine** [comment]
FWIW the effect on the total height of the media-body is pretty much a wash, as the screenshots show.  Short values like format take an extra line, while long ones get much shorter.

**Comment by maccabeelevine** [comment]
Just a second example, with an EDS record, to see how ridiculous the wrapping can get.

image

image

**Review by crhallberg — APPROVED** [review]
This is a brilliant change. Long overdue now that I've seen it!

**Comment by sturkel89** [comment]
This looks great in all three themes on mobile, and in Android as well as iOS. (I have a simulator plugin for my browser.) I think it's ready to merge!

**Comment by demiankatz** [comment]
Thanks, everyone!

**Pull request merged**